src/v/cluster/topics_frontend.h

```diff
@@ -180,6 +183,20 @@ class topics_frontend {
       do_cancel_moving_partition_replicas(
         std::vector<model::ntp>, model::timeout_clock::time_point);
 
+    struct health_monitor_info {
```

Comment:
Maybe a more specific name for this struct, like `capacity_info`, to make it more obvious what it's for.